### ISSUE TITLE
How do I use Torchvision R50 Weights

### ISSUE THREAD

**Issue by sarmientoj24** [microsoft/SoftTeacher#110]
I would want to use ViSSL and pre-train the weights before using SoftTeacher. However, I am having problems with the Caffe weights. I would like to know how do I use the Resnet-50 torchvision weights from here https://github.com/open-mmlab/mmdetection/blob/master/docs/model_zoo.md in this code. Or if it is possible.

**Comment by MendelXu** [comment]
You can just change the `style` in the model config to `pytorch` and write the path of the model in the `init_cfg`.
```
model = dict(
  
    backbone=dict(
        style='pytorch',
        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),
...
```

**Comment by zhaogev5** [comment]
I also use the torch checkpoint but performance is bad,even the loss is Nan,How can I solve it

**Comment by NimaDL** [comment]
I have got the same bad results using pytorch style. Is it required to change something else in the config file once the stype is changed to pytorch. By checking the visualization in wandb, the model estimation for bbox is really bad during the training phase.

**Comment by MendelXu** [comment]
Could anyone share your config file here?
